examples/README.md

```diff
@@ -8,8 +8,9 @@ the logic of the contract is reasonable, or safe.
 **Do not use these contracts as-is for anything other then experimenting.**
 
```

Comment:
Can you add the smart contracts to the CI pipeline please. This will run the test and compile the smart contracts to make sure they are working correctly.

It can be done by adjusting this file:
`.github/workflows/linter.yml`